themes/default/content/events-workshops/pulumi-update-2020-04-29/index.md

```diff
@@ -8,10 +8,11 @@ meta_image: "/images/webinar/pulumi-update-twitter-card.png"
 featured: false
 
 aliases:
-    - /webinars/pulumi-v2-virtual-launch
-    - /events/launch-v2-seattle-2020-04-15
-    - /webinars/pulumi-update-2020-04-22
-    - /webinars/pulumi-update-2020-04-29
+  - /resources/pulumi-update-2020-04-29
+  - /webinars/pulumi-v2-virtual-launch
+  - /events/launch-v2-seattle-2020-04-15
+  - /webinars/pulumi-update-2020-04-22
+  - /webinars/pulumi-update-2020-04-29
```

Comment:
I changed the indentation here to match what we are using for lists. Calling this out to not miss the addition of the alias `/resources/pulumi-update-2020-04-29` as the first item here.